test/lib/commands/version.js

```diff
@@ -184,13 +175,7 @@ t.test('empty versions', async t => {
       await version.exec([])
       t.same(
         result(),
-        [
-          {
-            'workspaces-test': '1.0.0',
-            'workspace-a': '1.0.0',
-            npm: '1.0.0',
-          },
-        ],
+        "{ 'workspace-a': '1.0.0', 'workspaces-test': '1.0.0', npm: '1.0.0' }",
```

Comment:
This is another interesting case of our usage of `util.format`. By default anything less than 80 characters will try to fit on a single line. This is customizable with `breakLength`. I don't think this is worth working on in this PR but we have the option to in the future since we use `util.formatWithOptions` now.

```
> console.log(util.formatWithOptions({breakLength: 1}, {a:1,b:[1,2,3]}))
{
  a: 1,
  b: [
    1,
    2,
    3
  ]
}

> console.log(util.formatWithOptions({breakLength: 80}, {a:1,b:[1,2,3]}))
{ a: 1, b: [ 1, 2, 3 ] }
```